server/src/main/java/org/elasticsearch/health/GetHealthAction.java

```diff
@@ -116,6 +124,22 @@ public String toString() {
     }
 
     public static class Request extends ActionRequest {
+        private final boolean includeDetails;
+        private final String componentName;
+        private final String indicatorName;
+
+        public Request(boolean includeDetails) {
+            this.includeDetails = includeDetails;
```

Comment:
is this flag needed? if component or indicator name is specified then details are requested I think (?)

Reply:
I think we shouldn't keep it until (and if) we add the functionality back as it could be confusing when inspecting the code.

Reply:
Done.